tests/testthat/indention_curly_brackets/custom-out.R

```diff
@@ -1,9 +1,6 @@
 value <- 5
-if (value > 0) {
-    print(value)
-}
-
+if (value > 0) print(value)
```

Comment:
Although we recommend side effects to be on their own line, we currently don't force it.

I wonder if we should force multi-line if the `if` is not assigned or part of a larger call though, since that indicates a side effect.